analyzer/src/main/kotlin/PackageManagerDependency.kt

```diff
@@ -55,11 +55,6 @@ data class PackageManagerDependency(
             "Could not find a project for the definition file '$definitionFile'."
         }
 
```

Comment:
Have you considered to make `GradleInspector` use `Gradle` as `managerName` ?

Reply:
Not really, as it does not solve the underlying problem: Due to the plugin mechanism there always could be "alternative" package managers that manage projects of types that other "core" package managers already manage. And I plan to introduce a "MavenInspector" in the foreseeable future as well.